RDVFSharp/Commands/General/Restat.cs

```diff
@@ -23,18 +23,18 @@ public async Task Execute(string character, IEnumerable<string> args, string cha
 
             if (fighter == null)
             {
-                throw new FighterNotRegistered(character);
+                Plugin.FChatClient.SendMessageInChannel("You are not registered. Please register with the bot first using the !register command. Example: !register 5 8 8 1 2", channel);
             }
 
             int[] statsArray;
 
-            try
+            
```

Comment:
You should definitely not remove the try/catch here! It can fail in many ways. Keep the "throw new Exception();" but in the catch, send the message instead of throwing another exception